src/components/30-organisms/commercial-hero-banner/story.js

```diff
@@ -0,0 +1,65 @@
+/* global document */
```

Comment:
Storybook with Chrome do not show anything. I see the Knobs and Readme, but on top the iframe is empty.

Reply:
der build ist nicht richtig gelaufen. jetzt gehts.